editor/selectors.js

```diff
@@ -8,48 +8,121 @@ import { first, last, get } from 'lodash';
 
+/**
+ * Returns the current editing mode.
+ *
+ * @param  {Object} state Global application state
+ * @return {String}       Editing mode
+ */
 export function getEditorMode( state ) {
 	return state.mode;
 }
 
+/**
+ * Returns true if the editor sidebar panel is open, or false otherwise.
+ *
+ * @param  {Object}  state Global application state
+ * @return {Boolean}       Whether sidebar is open
+ */
 export function isEditorSidebarOpened( state ) {
 	return state.isSidebarOpened;
 }
 
+/**
+ * Returns true if the past editor history snapshots exist, or false otherwise.
+ *
+ * @param  {Object}  state Global application state
+ * @return {Boolean}       Whether undo history exists
+ */
 export function hasEditorUndo( state ) {
 	return state.editor.history.past.length > 0;
 }
 
+/**
+ * Returns true if the future editor history snapshots exist, or false
+ * otherwise.
+ *
+ * @param  {Object}  state Global application state
+ * @return {Boolean}       Whether redo history exists
+ */
 export function hasEditorRedo( state ) {
 	return state.editor.history.future.length > 0;
 }
 
+/**
+ * Returns true if the currently edited post is yet to be saved, or false if
+ * the post has been saved.
+ *
+ * @param  {Object}  state Global application state
+ * @return {Boolean}       Whether the post is new
+ */
 export function isEditedPostNew( state ) {
 	return ! state.currentPost.id;
 }
 
+/**
+ * Returns true if there are unsaved values for the current edit session, or
+ * false if the editing state matches the saved or new post.
+ *
+ * @param  {Object}  state Global application state
+ * @return {Boolean}       Whether unsaved values exist
+ */
 export function isEditedPostDirty( state ) {
 	return state.editor.dirty;
 }
 
+/**
+ * Returns the post currently being edited.
+ *
+ * @param  {Object} state Global application state
+ * @return {Object}       Post object
+ */
 export function getCurrentPost( state ) {
 	return state.currentPost;
 }
 
+/**
+ * Returns the post values which have yet to be saved.
+ *
+ * @param  {Object} state Global application state
+ * @return {Object}       Object of key value pairs comprising unsaved edits
+ */
 export function getPostEdits( state ) {
 	return state.editor.edits;
 }
 
+/**
+ * Returns a single attribute of the post being edited.
+ *
+ * @param  {Object} state         Global application state
+ * @param  {String} attributeName Post attribute name
+ * @return {*}                    Post attribute value
+ */
 export function getEditedPostAttribute( state, attributeName ) {
 	return state.editor.edits[ attributeName ] === undefined
 		? state.currentPost[ attributeName ]
 		: state.editor.edits[ attributeName ];
 }
 
+/**
+ * Returns the current status of the post being edited, preferring the unsaved
+ * value if different than the saved post.
+ *
+ * @param  {Object} state Global application state
+ * @return {String}       Post status
+ */
 export function getEditedPostStatus( state ) {
```

Comment:
This has been written before the `getEditedPostAttribute`, I wonder if we should just drop it now.